netbox/utilities/templatetags/form_helpers.py

```diff
@@ -32,6 +34,15 @@ def getfield(form, fieldname):
         return None
 
 
+@register.filter()
+def getfilterfield(form, fieldname):
```

Comment:
We should rename this for readability. Also please add a docstring explaining its function.
```suggestion
def get_filter_field(form, fieldname):
```